internal/builders/gradle/action.yml

```diff
@@ -47,6 +47,10 @@ outputs:
       Users should verify the download against this digest to prevent tampering.
     value: ${{ steps.upload-build-dir.outputs.sha256 }}
 
+  package-download-name:
```

Comment:
should this be called `build-download-name` for consistency?

Reply:
Should have fixed all names.